workflow/applications.py

```diff
@@ -409,9 +409,6 @@ def _get_cycled_task_names(self):
         if self.do_metp:
             gfs_tasks += ['metp']
 
-        if self.do_hybvar and 'gfs' in self.eupd_cdumps:
```

Comment:
Is this really the correct solution? If these jobs shouldn't run for gfs, then I think they should be moved to the hybrid_gdas_tasks list instead of hybrid_gdas_or_gfs_tasks. But I'm not 100% sure we don't need these jobs for early cycle. @aerorahul?